src/components/DropDown.tsx

```diff
@@ -3,26 +3,34 @@ import React from "react";
 import { CustomDropdownProps } from "../@types/components/commonTypes";
 
 const CustomDropdown: React.FC<CustomDropdownProps> = (props) => {
+  const {
+    containerclassname = "",
+    labelclassname = "",
+    imageclassname = "",
+    className = "",
+  } = props;
+
   return props.icon ? (
     <div
-    className={`w-full self-end ${props.containerclassname}`}
+      className={`w-full self-end ${containerclassname}`}
```

Comment:
${containerclassname || ""}
 please apply this for the subsequent scenarios

Reply:
default prop is already added as ""